kernel-modules/BLOCKLIST

```diff
@@ -72,8 +72,10 @@
 6.2.* 2.2.0 *
 6.2.* 2.1.0 *
 6.2.* 2.0.1 *
-# TODO(ROX-16705) - 6.3/6.4 kernel compilation errors
+# ROX-16705 - 6.3/6.4 kernel compilation errors
 ~6\.[3-9]\.\d+.* ~2\.[2-5]\.\d+(?:-rc\d+)?
+# TODO(ROX-20541) - 6.6 kernel compilation errors
+6.6.* *
```

Comment:
You don't need the second parameter as a `*`, since the script that processes this file assumes all missing parts are `*`. https://github.com/stackrox/collector/blob/24b43e3f04594741fc92957662a4eae41bc53fdb/kernel-modules/build/apply-blocklist.py#L41C8-L41C8

Reply:
And it is explained in the comment header... Removed it !